server/tests/unit/data_common/fbs/test_matrix.py

```diff
@@ -46,7 +46,7 @@ def test_encode_DataFrame(self):
                 "d": pd.Series(["x", "y", "z", "x", "y", "z", "a", "x", "y", "z"], dtype="category"),
```

Comment:
See comment on common/fbs/test_matrix.py about needing to test the new features/encodings.